config/genesis/genesis.go

```diff
@@ -0,0 +1,35 @@
+package genesis
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"path/filepath"
+)
+
+type Genesis struct {
+	Name       string
+	Id         string
+	Bootnodes  []string
```

Comment:
Why do we need boot nodes here?

Reply:
I'm going off of polkadot's genesis file: https://raw.githubusercontent.com/paritytech/polkadot/master/service/res/kusama.json I think it makes sense since bootnodes aren't going to change over the lifetime of the chain

Reply:
Yeah it should be in the `genesis.json` file I am just unsure if it needs to be included in the `genesis` struct

Reply:
the genesis struct is what the json is unmarshalled to so it has to be there